docs/java-rest/high-level/getting-started.asciidoc

```diff
@@ -154,3 +154,23 @@ executes the request. For example, this is the place where you'd specify a
 `NodeSelector` to control which node receives the request. See the
 <<java-rest-low-usage-request-options,low level client documentation>> for
 more examples of customizing the options.
+=== Asynchronous usage
+
+All of the the methods accross the different clients exist in a traditional synchronous and 
+asynchronous variant. The difference is that the asynchronous ones use asynchrous apache 
+httpclient requests. This is useful if you are doing multiple requests or are using e.g.
+rx java, Kotlin co-routines, or similar frameworks.
+
+The asynchrouns calls are recognizable by the fact that they have the word Async in their name 
+and return a Cancellable instance. The asynchronous methods accept the same request object 
+as the synchronous variant and accept a generic `ActionListener<T>` where `T` is the return 
+type of the synchronous method. 
+
+All asynchronous methods return a `Cancellable` object with a cancel method that you may call 
+in case of e.g. an error or in case you wish to abort the request. Cancelling
+no longer needed requests is a good way too avoid putting unnecessary 
+load on Elasticsearch.
+
+Using the `Cancellable` instance is optional and you can safely ignore this if you have 
+no need for this. A use case for this would be using this with e.g. Kotlin's `suspendCancellableCoRoutine`. 
```

Comment:
I feel like thee paragraphs are repeating what we already have here in other words: https://www.elastic.co/guide/en/elasticsearch/client/java-rest/master/java-rest-low-usage-requests.html#_cancelling_asynchronous_requests 

any particular bit that you think is missing there and would be worth adding?

Reply:
sorry I did not notice that this is for the high-level client. this was for the other page you have changed.